### DISCUSSION TITLE
KEDA scale from/to 0 limitation

### DISCUSSION THREAD

**Discussion by Jeffwan** [kedacore/keda#1840] [Q&A / Need Help] [answered]
Trying to confirm the scaling limitation. 

1. If the metrics is from external, which is not related to the scale target itself. I think scale from/to 0 works well
2. As KEDA also support prometheus, Let's say we want to use QPS as the metrics, then I think scale up from 0 won't work?  Because QPS comes from application itself, metrics will be unknown after scaling down to 0.

**Comment by tomkerkhove** [comment, marked as answer]
Scale to/from 0 is optional as you can define the minimum replica count.

I don't have experience with QPS but if it comes from the containers themselves it will indeed not work, unfortunately.

**Reply by Jeffwan** [reply]
Thanks! Since KEDA concentrates on external metrics, I think it is acceptable. The metrics used to trigger scale up/down mostly comes from external and doesn't reply on application itself.  

I will check other solutions to see if I can activate the pods based on application metrics.

**Reply by tomkerkhove** [reply]
Just out of curiosity, how do you expect there to be app metrics, if they are scaled to 0? 😅

**Comment by hitsub2** [comment]
Is that possible to have multiple triggers, with one trigger settting activationThreshold to 0?
